tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,3330 @@
+            latestMilestone:
+              type: object
+              description: Information about the latest seen milestone.
+              properties:
+                index:
+                  type: integer
+                  description: The index of the latest seen milestone.
+                timestamp:
+                  type: integer
+                  description: The timestamp of the latest seen milestone.
+                milestoneId:
+                  type: string
+                  description: The Milestone ID of the latest seen milestone. Hex-encoded with 0x prefix.
+              required:
+                - index
+            confirmedMilestone:
+              type: object
+              description: Information about the latest confirmed milestone.
+              properties:
+                index:
+                  type: integer
+                  description: The index of the latest confirmed milestone.
+                timestamp:
+                  type: integer
+                  description: The timestamp of the latest confirmed milestone.
+                milestoneId:
+                  type: string
+                  description: The Milestone ID of the latest confirmed milestone. Hex-encoded with 0x prefix.
+              required:
+                - index
+          required:
+            - isHealthy
+            - latestMilestone
+            - confirmedMilestone
+        metrics:
+          description: Node metrics.
+          properties:
+            blocksPerSecond:
+              description: The current rate of new blocks per second.
+              type: number
+              format: float
+            referencedBlocksPerSecond:
+              description: The current rate of referenced blocks per second.
+              type: number
+              format: float
+            referencedRate:
+              description: The ratio of referenced blocks in relation to new blocks of the last confirmed milestone.
+              type: number
+              format: float
+          required:
+            - blocksPerSecond
+            - referencedBlocksPerSecond
+            - referencedRate
+        supportedProtocolVersions:
+          type: array
+          description: The supported protocol versions.
+          items:
+            type: integer
+        protocol:
+          description: Protocol parameters.
+          properties:
+            networkName:
+              type: string
+              description: The Name of the network from which the networkId is derived.
+            bech32Hrp:
+              type: string
+              description: Tells whether the node supports mainnet or testnet addresses. Value `iota` indicates that the node supports mainnet addresses. Value `atoi` indicates that the node supports testnet addresses.
+            tokenSupply:
+              type: string
+              description: Current supply of base token. Plain string encoded number.
+            protocolVersion:
```

Comment:
In the current node response this field is actually `version` (which imo is the better name).
The iota.js implementation follows this spec, so it won't work atm (undefined).
So we either fix iota.js and the TIP, or node api response.

Reply:
will fix this